lib/Controller/TargetController.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace OCA\Richdocuments\Controller;
+
+use OCA\Richdocuments\Service\FileTargetService;
+use OCP\AppFramework\Http;
+use OCP\AppFramework\Http\DataDisplayResponse;
+use OCP\AppFramework\Http\DataResponse;
+use OCP\AppFramework\Http\Response;
+use OCP\IRequest;
+
+class TargetController extends \OCP\AppFramework\OCSController {
+
+	public function __construct($appName, IRequest $request, $corsMethods = 'PUT, POST, GET, DELETE, PATCH', $corsAllowedHeaders = 'Authorization, Content-Type, Accept', $corsMaxAge = 1728000,
+		private FileTargetService $fileTargetService,
```

Comment:
Optional parameter is provided before required. Either move `FileTargetService` to the beginning or make it optional as well. Also, the line probably could be formatted better :wink: